deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -488,14 +519,73 @@ handle_tick(QName,
                                                        rabbit_fifo:make_purge_nodes(Stale)),
 
                               ok
-                      end
+                      end,
+
+                      %% ----------
+                      maybe_grow_qq_members(QName, LastActive)
                   catch
                       _:_ ->
                           ok
                   end
           end),
     ok.
 
+maybe_grow_qq_members(QName, LastActive) ->
+    {ok, Q} = rabbit_amqqueue:lookup(QName),
+    % Rely on LastActive and compare it with system_time, to figure out intervall
+    case check_if_grow(LastActive) of
+        false ->
+            rabbit_log:info(">>>> TICK do nothing",[]),
+            ok;
+        true ->
+            rabbit_log:info(">>>> TICK check members",[]),
+            Running = rabbit_nodes:all_running(),
+            {ok, Members, _} = ra:members(amqqueue:get_pid(Q)),
```

Comment:
@kjnilsson Using ra:members() instead of type_state. But if we could get this info in the tick, all for that!